homeassistant/components/alert/__init__.py

```diff
@@ -48,7 +48,12 @@
         vol.Required(CONF_NAME): cv.string,
         vol.Required(CONF_ENTITY_ID): cv.entity_id,
         vol.Required(CONF_STATE, default=STATE_ON): cv.string,
-        vol.Required(CONF_REPEAT): vol.All(cv.ensure_list, [vol.Coerce(float)]),
+        vol.Required(CONF_REPEAT): vol.All(
+            cv.ensure_list,
+            [vol.Coerce(float)],
+            # Minimum delay is 1 second = 0.16 minutes
```

Comment:
Is it 0.16 (as in the comment) or 0.016 as in the code ?

I think 0.016 is the right number.

Reply:
```suggestion
            # Minimum delay is 1 second = 0.016 minutes
```